manifests/install/repos/extra.pp

```diff
@@ -30,12 +30,7 @@
   }
 
   if $ipa_authentication and $configure_ipa_repo {
-    yumrepo { 'adelton-identity':
-      enabled  => 1,
-      gpgcheck => 0,
-      baseurl  => "http://copr-be.cloud.fedoraproject.org/results/adelton/identity_demo/epel-${osreleasemajor}-\$basearch/",
-      before   => Package['mod_authnz_pam', 'mod_lookup_identity', 'mod_intercept_form_submit', 'sssd-dbus'],
-    }
+    notify {'The --foreman-configure-ipa-repo option is deprecated and no longer needed. If you have used it before, disable and delete the repo.':}
```

Comment:
> but rather class parameter

Do we have many users that knowingly use these classes and class parameters? My guess would be that the vast majority would hit this code via `foreman-install` command.

OTOH, if the installer has some mechanism to mark the options as obsolete and notify the user even before running puppet, that might be the best course of action.